tools/gen-stage-test-diff

```diff
@@ -8,7 +8,7 @@ import tempfile
 
 def run_osbuild(output_directory, store, cache_max_size, libdir, manifest):
     args = [
-        "osbuild",
+        "python3", "-m", "osbuild",
```

Comment:
Please use `sys.executable` instead of `python3`.

Reply:
Nice catch and excellent point. I fixed it here and in `test/test.py` now.